source/_integrations/smappee.markdown

```diff
@@ -39,3 +46,9 @@ client_secret:
   required: true
   type: string
 {% endconfiguration %}
+
+<div class='note'>
+
+If Home Assistant is running in an non local environment, you must make sure that you have configured the `internal_url` and `external_url` in your [configuration](/docs/configuration/basic) in order to complete to OAuth2 configuration flow.
```

Comment:
This note is incorrect. OAuth2 only relies on the internal URL, when not set, it will use the IP address detected of the main network interface, combined with the port and SSL settings of the `http` integration.

The external URL has no effect on the OAuth2 configuration flow.